crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -64,6 +64,15 @@ def upgrade!
             message: msg
           }
         end
+      rescue StandardError => e
```

Comment:
This will also catch StartStepRunningError etc. which should bubble up to the controller. If we leave this here, `start_step()` when status is `running` will turn the status to `failed`.